types/synetica/enlinkStatusDP/meta.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "Enlink DP",
+  "name": "enLink Status-DP",
```

Comment:
changes in name will need to be done manually, therefore coordination when merging this is needed